app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -531,16 +508,14 @@ class OnboardingFragmentTest {
   }
 
   @Test
-  // TODO(#973): Fix OnboardingFragmentTest
-  @Ignore
   fun testOnboardingFragment_clickOnSkip_changeOrientation_titleIsCorrect() {
     launch(OnboardingActivity::class.java).use {
```

Comment:
@MohamedMedhat1998 @rt4914 Now only this test is failing

Reply:
@prayutsu Can you mention the error?

Reply:
@rt4914 actually it is recognising the text of first onboarding slide i.e - `Welcome to Oppia!` 


```
androidx.test.espresso.base.DefaultFailureHandler$AssertionFailedWithCauseError: 'with string from resource id: <2131755271>' doesn't match the selected view.
Expected: with string from resource id: <2131755271>[onboarding_slide_3_title] value: Have fun!
     Got: "AppCompatTextView{id=2131296884, res-name=slide_title_text_view, visibility=VISIBLE, width=792, height=41, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=androidx.constraintlayout.widget.ConstraintLayout$LayoutParams@12174f72, tag=null, root-is-layout-requested=false, has-input-connection=false, x=84.0, y=888.0,


 **text=Welcome to Oppia!, **

input-type=0, ime-target=false, has-links=false}"
```

Reply:
> @rt4914 actually it is recognising the text of first onboarding slide i.e - `Welcome to Oppia!`
> 
> I have highlighted the text being recognised (**)
> 
> ```
> androidx.test.espresso.base.DefaultFailureHandler$AssertionFailedWithCauseError: 'with string from resource id: <2131755271>' doesn't match the selected view.
> Expected: with string from resource id: <2131755271>[onboarding_slide_3_title] ***value: Have fun!***
>      Got: "AppCompatTextView{id=2131296884, res-name=slide_title_text_view, visibility=VISIBLE, width=792, height=41, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=androidx.constraintlayout.widget.ConstraintLayout$LayoutParams@12174f72, tag=null, root-is-layout-requested=false, has-input-connection=false, x=84.0, y=888.0,
> 
> 
>  **text=Welcome to Oppia!, **
> 
> input-type=0, ime-target=false, has-links=false}"
> ```


The text of the third slide (`Have fun!`) should be recognized and not that of first slide (`Welcome to Oppia!`)